scripts/config.py

```diff
@@ -202,6 +202,8 @@ def realfull_adapter(name, value, _active):
         return True
     if name == 'MBEDTLS_CONFIG_VERSION':
         return True
+    if name.startswith('PSA_WANT_'):
+        return True
```

Comment:
Shouldn't we use the `is_boolean_setting()` function introduced a few commits earlier? We're doing the same here (`if not value` then `if name.startswith('PSA_WANT_')`.

Reply:
This piece of code no longer exists after the complete redesign of `realfull`.